src/main/test/edu/fiuba/algo3/modelo/cards/specials/ScorchTest.java

```diff
@@ -0,0 +1,121 @@
+
+        rangedRow.placeCard(unidad1);
+        rangedRow.placeCard(unidad2);
+        tierraArrasada.play(specialZone);
+
+        assertFalse(rangedRow.getCards().contains(unidad2));
+    }
+
+    @Test
+    public void testSeJuegaUnaTierraArrasadaSeEliminaLaCartaMasFuerteDeLaFilaSiegeCorrectamente() {
+        Unit unidad1 = new Unit("Nombre", "Descripcion", 4, new SiegeType(), List.of(new TightBond()));
+        Unit unidad2 = new Unit("Nombre", "Descripcion", 6, new SiegeType(), List.of(new MoraleBoostModifier()));
+        Unit unidad3 = new Unit("Nombre", "Descripcion", 8, new SiegeType(), List.of());
+
+        siegeRow.placeCard(unidad1);
+        siegeRow.placeCard(unidad2);
+        siegeRow.placeCard(unidad3);
+        tierraArrasada.play(specialZone);
+
+        assertFalse(siegeRow.getCards().contains(unidad3));
+    }
+
+    @Test
+    public void testSeJuegaUnaTierraArrasadaSeQuemanLasCartasMasFuertesDeLaFilaCloseCombat() {
+        Unit unidad1 = new Unit("Nombre", "Descripcion", 6, new CloseCombatType(), List.of(new TightBond()));
+        Unit unidad2 = new Unit("Nombre", "Descripcion", 4, new CloseCombatType(), List.of(new MoraleBoostModifier()));
+        Unit unidad3 = new Unit("Nombre", "Descripcion", 6, new CloseCombatType(), List.of());
+
+        closeCombatRow.placeCard(unidad1);
+        closeCombatRow.placeCard(unidad2);
+        tierraArrasada.play(specialZone);
+
+        assertFalse(closeCombatRow.getCards().contains(unidad1));
+        assertFalse(closeCombatRow.getCards().contains(unidad3));
+    }
+
+    @Test
+    public void testSeJuegaUnaTierraArrasadaSeQuemanLasCartasMasFuertesDeLaFilaRanged() {
+        Unit unidad1 = new Unit("Nombre", "Descripcion", 4, new RangedType(), List.of(new TightBond()));
+        Unit unidad2 = new Unit("Nombre", "Descripcion", 8, new RangedType(), List.of(new MoraleBoostModifier()));
+        Unit unidad3 = new Unit("Nombre", "Descripcion", 8, new RangedType(), List.of());
+
+        rangedRow.placeCard(unidad1);
+        rangedRow.placeCard(unidad2);
+        tierraArrasada.play(specialZone);
+
+        assertFalse(rangedRow.getCards().contains(unidad2));
+        assertFalse(rangedRow.getCards().contains(unidad3));
+    }
+
+    @Test
+    public void testSeJuegaUnaTierraArrasadaSeEliminaLasCartasMasFuertesDeLaFilaSiege() {
+        Unit unidad1 = new Unit("Nombre", "Descripcion", 8, new SiegeType(), List.of(new TightBond()));
+        Unit unidad2 = new Unit("Nombre", "Descripcion", 8, new SiegeType(), List.of());
+        Unit unidad3 = new Unit("Nombre", "Descripcion", 6, new SiegeType(), List.of(new MoraleBoostModifier()));
+
+        siegeRow.placeCard(unidad1);
+        siegeRow.placeCard(unidad2);
+        siegeRow.placeCard(unidad3);
+        tierraArrasada.play(specialZone);
+
+        assertFalse(siegeRow.getCards().contains(unidad1));
+        assertFalse(siegeRow.getCards().contains(unidad2));
+    }
+
```

Comment:
Missing case: Remove Strongest cards from both players
```suggestion

    @Test
    public void testSeJuegaUnaTierraArrasadaSeEliminaLasCartasMasFuertesDeAmbosPlayers() {
        Unit unidad1 = new Unit("Nombre", "Descripcion", 8, new RangedType(), List.of());
        Unit unidad2 = new Unit("Nombre", "Descripcion", 8, new SiegeType(), List.of());
        Unit unidad3 = new Unit("Nombre", "Descripcion", 4, new RangedType(), List.of());
        Unit unidad4 = new Unit("Nombre", "Descripcion", 6, new SiegeType(), List.of());

        player1RangedRow.placeCard(unidad1);
        player2SiegeRow.placeCard(unidad2);
        player1RangedRow.placeCard(unidad3);
        player2SiegeRow.placeCard(unidad4);
        specialZone.placeCard(tierraArrasada);

        assertFalse(player1RangedRow.getCards().contains(unidad1));
        assertFalse(player2SiegeRow.getCards().contains(unidad2));
    }
```